examples/react-presence/.env

```
@@ -0,0 +1,2 @@
+VITE_YORKIE_API_ADDR='http://localhost:8080'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)